libs/spark/src/NavBar.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { AppBar, AppBarProps, withStyles } from '@material-ui/core/';
+import { palette } from './styles/palette';
+
+export interface NavBarProps extends AppBarProps {
+  color?: 'default';
+}
+
+const StyledAppBar = withStyles({
+  root: {
+    borderBottom: `2px solid ${palette.grey.medium}`,
+    fontSize: '1.125rem',
+    color: palette.text.onLightLowContrast,
+  },
+})(AppBar);
+
+export const NavBar = ({ color = 'default', ...other }: NavBarProps) => {
+  return (
+    <StyledAppBar position="fixed" elevation={0} color={color} {...other} />
```

Comment:
I think we should also set `component="nav"`, the [landmark HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).

Reply:
Added ✅